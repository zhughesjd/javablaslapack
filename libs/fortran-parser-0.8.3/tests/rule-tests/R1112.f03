
! Testing only list
use mymodule , only : this, that, theother
end
