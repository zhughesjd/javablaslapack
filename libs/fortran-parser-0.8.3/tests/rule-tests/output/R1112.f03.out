1101:begin
1112:list-begin
1112:list 3
1103
1101 false false false
